RxTodo/Sources/ViewControllers/TaskListViewController.swift

```diff
@@ -14,109 +14,110 @@ import RxSwift
+          make.edges.equalTo(0)
+        }
     }
 
-    // Input
-    self.addBarButtonItem.rx.tap
-      .bindTo(viewModel.addButtonDidTap)
-      .addDisposableTo(self.disposeBag)
-
-    self.tableView.rx.itemSelected
-      .bindTo(viewModel.itemDidSelect)
-      .addDisposableTo(self.disposeBag)
-
-    self.tableView.rx.itemDeleted
-      .bindTo(viewModel.itemDeleted)
-      .addDisposableTo(self.disposeBag)
-
-    // Ouput
-    viewModel.navigationBarTitle
-      .drive(self.navigationItem.rx.title)
-      .addDisposableTo(self.disposeBag)
-
-    viewModel.sections
-      .drive(self.tableView.rx.items(dataSource: self.dataSource))
-      .addDisposableTo(self.disposeBag)
-
-    viewModel.presentTaskEditViewModel
-      .drive(onNext: { [weak self] viewModel in
-        guard let `self` = self else { return }
-        let viewController = TaskEditViewController(viewModel: viewModel)
-        let navigationController = UINavigationController(rootViewController: viewController)
-        self.present(navigationController, animated: true, completion: nil)
-      })
-      .addDisposableTo(self.disposeBag)
-  }
-
+    // MARK: Configuring
+
+    private func configure(_ viewModel: TaskListViewModelType) {
+        dataSource.configureCell = { _, tableView, indexPath, viewModel in
+          let cell = tableView.dequeue(Reusable.taskCell, for: indexPath)
+          cell.configure(viewModel)
+          return cell
+        }
+
+        // Input
+        addBarButtonItem.rx.tap
+          .bindTo(viewModel.addButtonDidTap)
+          .addDisposableTo(disposeBag)
+
+        tableView.rx.itemSelected
+          .bindTo(viewModel.itemDidSelect)
+          .addDisposableTo(disposeBag)
+
+        tableView.rx.itemDeleted
+          .bindTo(viewModel.itemDeleted)
+          .addDisposableTo(disposeBag)
+
+        // Ouput
+        viewModel.navigationBarTitle
+          .drive(navigationItem.rx.title)
+          .addDisposableTo(disposeBag)
+
+        viewModel.sections
+          .drive(tableView.rx.items(dataSource: dataSource))
+          .addDisposableTo(disposeBag)
+
+        viewModel.presentTaskEditViewModel
+          .drive(onNext: { [weak self] viewModel in
+            let viewController = TaskEditViewController(viewModel: viewModel)
+            let navigationController = UINavigationController(rootViewController: viewController)
+            self?.present(navigationController, animated: true, completion: nil)
+          })
+          .addDisposableTo(disposeBag)
+    }
 }
 
 
 // MARK: - UITableViewDelegate
 
 extension TaskListViewController: UITableViewDelegate {
-
-  func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-    let viewModel = self.dataSource[indexPath]
-    return TaskCell.height(fits: tableView.width, viewModel: viewModel)
-  }
-
-  func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-    tableView.deselectRow(at: indexPath, animated: true)
-  }
-
```

Comment:
I moved tableView(_tableView:, didSelectRowAt: ) method up to line 58 of this file using rx.itemSelected!